scripts/examples/gn_silabs_example.sh

```diff
@@ -322,6 +322,14 @@ else
         optArgs+="openthread_root=\"$GSDK_ROOT/util/third_party/openthread\" "
     fi
 
+    if [ "$USE_DOCKER" == true ] && [ "$USE_WIFI" == true ]; then
+        echo "Switching WiseConnect 2 SDK ROOT"
+        optArgs+="wiseconnect_sdk_root=\"$WISECONNECT_SDK_ROOT\" "
```

Comment:
could this not be added in the prefix parsing of --docker like is done for efr32?

```
--docker)
                optArgs+="efr32_sdk_root=\"$GSDK_ROOT\" "
```